app/assets/javascripts/admin/addon/components/admin-config-areas/color-palettes.gjs

```diff
@@ -13,31 +13,28 @@ export default class AdminConfigAreasColorPalettes extends Component {
   @service router;
   @service modal;
 
-  get baseColorPalettes() {
-    return this.args.palettes.filter((palette) => palette.is_base);
-  }
-
   @action
   newColorPalette() {
     this.modal.show(ColorSchemeSelectBaseModal, {
       model: {
-        baseColorSchemes: this.baseColorPalettes,
+        colorSchemes: this.args.palettes,
         newColorSchemeWithBase: this.newColorPaletteWithBase,
       },
     });
   }
 
   @action
   async newColorPaletteWithBase(baseKey) {
-    const base = this.baseColorPalettes.find(
-      (palette) => palette.base_scheme_id === baseKey
-    );
+    const base = this.args.palettes.findBy("id", baseKey);
```

Comment:
These array prototype extension methods (`findBy`, `mapBy` etc.) are deprecated and will be removed soon from Ember. We should stick to `find()`.